services/app-api/utils/data/data.ts

```diff
@@ -49,7 +57,8 @@ export const removeNotApplicablePopsFromInitiatives = (
     initiative.defineInitiative_targetPopulations =
       initiative.defineInitiative_targetPopulations?.filter(
         (initiativePopulation: AnyObject) =>
-          !notApplicablePopulationNames.includes(initiativePopulation.value)
+          !notApplicablePopulationNames.includes(initiativePopulation.value) ||
```

Comment:
This scares me! Someone tell me if I need to check other impact from this change...

Reply:
@jessabean Is `notApplicablePopulationNames` the `other` custom target population names that have `No ` selected?

Reply:
Took me a while to remember what Not Applicable did lol, it should be fine. Not applicable I think is the fallback if no populations are shown.

Reply:
I believe the `notApplicablePopulationNames` is the filtered list of target populations that have "no" checked for being applicable to the initiative. Previously we didn't want to show these on the initiative page. Now we are going to continue to filter them out _except_ if they are considered a default population.